### PR TITLE
Remove IDB tool

### LINKED ISSUE #1899: [Tool] Link to IDB is dead
**Platform:**
iOS

**Description:**

The link to IDB homepage  in [this section](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x08-Testing-Tools.md#idb) is dead:

image

However, the [GitHub repo](https://github.com/dmayer/idb) still exists even if it was not updated since 2017.

Thanks a lot in advance 😃

### PR DESCRIPTION
**Pull request by righettod**
Hi,

This PR closes #1899 .

Thanks a lot in advance 😃

### CODE REVIEW

**Comment by righettod** [comment]
Link check failed but it is not due to the PR because I have only removed content 😢

File: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md [hunk 1]
```diff
@@ -695,12 +695,6 @@ Below are procedures for checking the binary security features described above.
     [SNIP]
     ```
 
-#### With idb
-
-[IDB](0x08-Testing-Tools.md#idb) automates the processes of checking for stack canary and PIE support. Select the target binary in the IDB GUI and click the "Analyze Binary…" button.
-
-<img src="Images/Chapters/0x06i/idb.png" alt="IDB Analyze Binary" width="350px" />
```
Comment (cpholguera) [inline Document/0x06i-Testing-Code-Quality-and-Build-Settings.md:702]: please also delete the image :)

**Review by cpholguera — CHANGES_REQUESTED** [review]
Thanks @righettod , only one thing should be still done.

**Comment by cpholguera** [comment]
Side note: I've edited your PR description so that the issue will be automatically closed once merged. Feel free to keep using this syntax in future PRs ;)

https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

**Comment by righettod** [comment]
@cpholguera Thanks a lot for the hint, I was not aware of this feature.

I cannot find the section in the file, am i missing something?

image

File: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md (hunk 1 above)
Comment (cpholguera) [reply Document/0x06i-Testing-Code-Quality-and-Build-Settings.md:702]: here's the path to the image to be removed. See the deleted line.

**Review by cpholguera — CHANGES_REQUESTED** [review]
Please see the comment. All images are in the Document/Images folder

File: Document/0x06h-Testing-Platform-Interaction.md [hunk 2]
```diff
@@ -2244,15 +2237,7 @@ OK!
...
-##### Using IDB
-
-In the **URL Handlers** section, go to the **Fuzzer** tab. On the left side default IDB payloads are listed. Once you have generated your payload list (e.g. using FuzzDB), go to the **Fuzz Template** section in the left bottom panel and define a template. Use `$@$` to define an injection point, for example:
-
-```bash
-myURLscheme://$@$
-```
-
-While the URL scheme is being fuzzed, watch the logs (see the section "[Monitoring System Logs](0x06b-Basic-Security-Testing.md#monitoring-system-logs "Monitoring System Logs")" of the chapter "iOS Basic Security Testing" to observe the impact of each payload. The history of used payloads is on the right side of the IDB "Fuzzer" tab.
+While the URL scheme is being fuzzed, watch the logs (see the section "[Monitoring System Logs](0x06b-Basic-Security-Testing.md#monitoring-system-logs "Monitoring System Logs")" of the chapter "iOS Basic Security Testing" to observe the impact of each payload.
```
Comment (cpholguera) [inline Document/0x06h-Testing-Platform-Interaction.md:2240]: I think this was part of the IDB section, it doesn't apply to the frida script
```suggestion
```

**Review by cpholguera — CHANGES_REQUESTED** [review]
one more thing I've noticed :)

File: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md (hunk 1 above)
Comment (righettod) [reply Document/0x06i-Testing-Code-Quality-and-Build-Settings.md:702]: I apologize for the disturbing, I was not really awaken 😖 
As the folder will become empty do you want that I add a `.gitkeep` file?

File: Document/0x06h-Testing-Platform-Interaction.md (hunk 2 above)
Comment (righettod) [reply Document/0x06h-Testing-Platform-Interaction.md]: OK i have commited your suggestion, you are right.

File: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md (hunk 1 above)
Comment (cpholguera) [reply Document/0x06i-Testing-Code-Quality-and-Build-Settings.md:702]: it's not necessary, thanks 😊

**Review by cpholguera — APPROVED** [review]
Thanks for the removal @righettod!

**Pull request merged**